apps/fumufumu-backend/src/index.ts

```diff
@@ -78,21 +81,19 @@ app.get('/health', async (c) => {
 })
 
 
-// ------------------------------------------
-// API ルーティング
-// ------------------------------------------
+// --- APIルーター（/api配下） ---
+const api = new Hono<{ Bindings: Env, Variables: Variables }>();
 
-// API グループを作成 (メインルーターとして機能)
-const api = new Hono<{ Bindings: Env, Variables: Variables }>()
+// カスタム認証エンドポイント（/api/auth/signup, /api/auth/signinなど）
+api.route('/auth', authRouter);
 
-// 認証ルーターを登録: /api/signup, /api/signin に対応
-api.route('/', authRouter);
-
-// 保護ルーターを登録: /api/protected に対応
+// 保護されたエンドポイント（/api/protected）
```

Comment:
私のコメントが悪いのですが、以下のコメントに変えておいていただけますと嬉しいです～
```suggestion
// 認証テスト用エンドポイント（/api/protected）
```